### PR TITLE
[script.video.bgtime_tv] 1.0.7

### PR DESCRIPTION
**Pull request by MiimMarkova**
### Checklist:

- [ x ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ x ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [ x ] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.video.bgtime_tv/gui.py [hunk 1]
```diff
@@ -382,7 +382,7 @@ def getListData(self, url, data=None, is_menu=None):
 		if const.LIVETV_PATH in cleaned_url and len(cleaned_url)> len(const.LIVETV_PATH):
 			expiration=datetime.timedelta(minutes=15)
 
-		const.CACHE.set(str(url+'_'+const.ADDON.getSetting('username')+'_'+const.ADDON.getSetting('password')), data, expiration=expiration)
+		const.CACHE.set(str(url+'_'+const.ADDON.getSetting('username')+'_'+const.ADDON.getSetting('password')), data as expiration=expiration)
```
Comment (Rechi) [inline script.video.bgtime_tv/gui.py]: that change doesn't look correct

File: script.video.bgtime_tv/addon.xml [hunk 2]
```diff
@@ -29,6 +28,6 @@
 		</disclaimer>
 		<website>https://bgtime.tv/</website>
 		<email>support@mytv.bg</email>
-		<news>v1.0.3 created by mariapmarkova@gmail.com</news>
+		<news>v1.0.7 created by mariapmarkova@gmail.com</news>
```
Comment (enen92) [inline script.video.bgtime_tv/addon.xml]: this is not the use for the news element :)
here you can simply share what's new in the latest version

File: script.video.bgtime_tv/addon.xml [hunk 3]
```diff
@@ -29,6 +28,6 @@
 		</disclaimer>
 		<website>https://bgtime.tv/</website>
 		<email>support@mytv.bg</email>
```
Comment (enen92) [inline script.video.bgtime_tv/addon.xml]: is this an official addon? If not I doubt mytv.bg will be happy to receive emails regarding the plugin

File: script.video.bgtime_tv/const.py [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+		self.rows = rows
+		self.cols = cols
+		self.filters = filters
+	def set_idx(self, idx):
+		self.idx = idx
+
+	def set_pos(self, x, y):
+		self.x = x
+		self.y = y
+		pass
```
Comment (enen92) [inline script.video.bgtime_tv/const.py]: minor: not needed

File: script.video.bgtime_tv/gui.py [hunk 5]
```diff
@@ -515,36 +432,66 @@ def getData(self, url):
...
+		data = self.makeUserPass()
+		self.token = controller.getData(const.SITE_LOGIN_PAGE, data)
+
+		if self.token:
+			self.writeInFile()
+			return True
+		return False
+	
+	def logOUT(self):
+		fopen 	= open(const.TOKEN_FILEPATH, "w+")
```
Comment (enen92) [inline script.video.bgtime_tv/gui.py]: you can use the `with` statement

**Pull request merged**